src/components/Tree/Tree.tsx

```diff
@@ -71,7 +71,11 @@ class Tree extends UIComponent<TreeProps> {
   }
 
   renderComponent({ ElementType, classes, accessibility, rest, styles, variables }) {
-    const { children } = this.props
+    const { children, nested } = this.props
+
+    if (nested) {
+      classes.root = `ui-subtree ${classes.root}`
```

Comment:
first point is that we should use `cx` function for that. Another is more about semantics of this branching - I don't agree that we should differentiate class names depending on whether it is a nested tree or not (classes are used for CSS purposes, so that `ui-tree ui-tree smt { .. }` selector would work just fine for subtree). Lets remove that and always use `ui-tree` class name, as it was before

Reply:
Done